leapp/cli/upgrade/__init__.py

```diff
@@ -211,11 +215,14 @@ def preupgrade(args):
 
     report_txt, report_json = [os.path.join(get_config().get('report', 'dir'),
                                             'leapp-report.{}'.format(f)) for f in ['txt', 'json']]
-    report_info([report_txt, report_json], fail=workflow.errors)
     # fetch all report messages as a list of dicts
     messages = fetch_upgrade_report_raw(context, renderers=False)
     with open(report_json, 'w+') as f:
         json.dump({'entries': messages}, f, indent=2)
+    report_files = [r for r in [report_txt, report_json]
+                    if os.path.isfile(r) and unix_timestamp_to_iso(os.path.getmtime(r)) > e.stamp]
```

Comment:
since old reports are archived, there is no need to check timestamp as you will always have only reports created in the current run

```suggestion
                    if os.path.isfile(r)]
```

Reply:
I have the same concern, why do we need timestamp filtering if reports are archived and the directory contains only the recent data?